docs/modules/ROOT/pages/how-tos/configuring/adding-custom-tasks.adoc

```diff
@@ -0,0 +1,146 @@
+= Adding custom tasks
+
+If your pipeline uses xref:advanced-how-tos/using-trusted-artifacts.adoc[Trusted Artifacts],
```

Comment:
We only talk about TA pipelines here. Can we have a section at the top describing the overall process and then two sections later discussing the uniqueness for TA and PVC pipeline tasks?